go.mod

```diff
@@ -84,3 +84,5 @@ require (
 	gotest.tools/v3 v3.5.1 // indirect
 	mvdan.cc/gofumpt v0.6.0 // indirect
 )
+
+replace github.com/gptscript-ai/chat-completion-client => github.com/njhale/chat-completion-client v0.0.0-20240430195304-47a4776457a1
```

Comment:
Note to self, remove replaces and update module after https://github.com/gptscript-ai/chat-completion-client/pull/1 merges.

Reply:
done